AdminWebsite/AdminWebsite.AcceptanceTests/Steps/SummarySteps.cs

```diff
@@ -261,8 +261,8 @@ private IEnumerable<HearingDetailsResponse> PollForHearingUpdated()
 
         private IEnumerable<HearingDetailsResponse> PollForAllHearings()
         {
-            const int RETRIES = 10;
-            const int DELAY = 2;
+            const int RETRIES = 20;
```

Comment:
That's a huge leap. Why 20?

Reply:
I was just trying to work out why the previous run failed. I have reversed it as I think I know why now